CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## master / unreleased
 
+* [CHANGE] Upgraded gogo/protobuf tool chain from 1.2.1 to 1.3.0. #2055
 * [CHANGE] Removed remaining support for using denormalised tokens in the ring. If you're still running ingesters with denormalised tokens (Cortex 0.4 or earlier, with `-ingester.normalise-tokens=false`), such ingesters will now be completely invisible to distributors and need to be either switched to Cortex 0.6.0 or later, or be configured to use normalised tokens. #2034
-* [CHANGE] Moved `--store.min-chunk-age` to the Querier config as `--querier.query-store-after`, allowing the store to be skipped during query time if the metrics wouldn't be found. The YAML config option `ingestermaxquerylookback` has been renamed to `query_ingesters_within` to match its CLI flag. #1893 
+* [CHANGE] Moved `--store.min-chunk-age` to the Querier config as `--querier.query-store-after`, allowing the store to be skipped during query time if the metrics wouldn't be found. The YAML config option `ingestermaxquerylookback` has been renamed to `query_ingesters_within` to match its CLI flag. #1893
```

Comment:
Could you rollback the changes here, except your entry, please? I think you have your editor trimming lines.

Reply:
My editor is linting markdown.

https://github.com/updownpress/markdown-lint/blob/master/rules/009-no-trailing-spaces.md

Reply:
Also currently the doc is inconsistent wrt spaces.

Reply:
Yeah, but this is not the PR where it should be fixed. Let's keep things compartmented.